kickstart/kickstart.json

```diff
@@ -1,15 +1,16 @@
 {
     "variables": {
-        "applicationId": "E9FDB985-9173-4E01-9D73-AC2D60D1DC8E",
-        "apiKey": "this_really_should_be_a_long_random_alphanumeric_value_but_this_still_works_dont_use_this_in_prod",
+        "apiKey": "33052c8a-c283-4e96-9d2a-eb1215c69f8f-not-for-prod",
         "asymmetricKeyId": "#{UUID()}",
-        "clientSecret": "this_really_should_be_a_long_random_alphanumeric_value_but_this_still_works_dont_use_this_in_prod",
+        "applicationId": "c2aeeb1e-65cd-43c5-9326-02dce5e03705",
```

Comment:
I was wandering what the groups opinion on this was. Should we all share the same sample application or should each quickstart have it's own application. I can see pros and cons of both.

Reply:
As kickstart will only be read when setting up a new instance, I think we can use the same Id to make things easier.

But I'd love if we had a feature to import a kickstart file into an existing instance, which would require us to have specific Ids for each application.

Reply:
I agree with @vcampitelli . I don't expect anyone to run multiple quickstart applications off the same FusionAuth server. 

We do have an open issue on importing a kickstart file into an existing instance.

See https://github.com/FusionAuth/fusionauth-issues/issues/560